lib/matplotlib/textpath.py

```diff
@@ -382,7 +383,10 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
                     charcode = glyph
 
                 if charcode is not None:
-                    glyph0 = font.load_char(charcode, flags=ft2font_flag)
+                    if use_glyph:
```

Comment:
This should probably be merged up into the conditionals above to simplify the logic?